server/channels/web/web_test.go

```diff
@@ -179,10 +181,11 @@ func (th *TestHelper) InitBasic() *TestHelper {
 	return th
 }
 
-func (th *TestHelper) TearDown() {
+func (th *TestHelper) TearDown(t *testing.T) {
```

Comment:
No objection to adding `testing.T` as an argument.

Alternatively, you can also add a `*testing.T` field  to the  `TestHelper` struct.

Reply:
Oh, thanks for the suggestion!